packages/ui-scripts/lib/utils/git.js

```diff
@@ -24,87 +24,31 @@
 const { runCommandSync, error, info } = require('@instructure/command-utils')
 const validateMessage = require('validate-commit-msg')
 
-const runGitCommand = (exports.runGitCommand = function runGitCommand(
-  args = []
-) {
+const runGitCommand = (exports.runGitCommand = (args = []) => {
   const { stdout } = runCommandSync('git', args, [], { stdio: 'pipe' })
   return stdout && stdout.trim()
 })
 
-exports.commit = function () {
-  try {
-    runGitCommand(['commit', '--dry-run'])
-  } catch (err) {
-    error(err.stdout)
-    process.exit(1)
-  }
-
-  try {
-    runCommandSync('yarn', ['husky:pre-commit'])
-  } catch (err) {
-    error(err.stdout)
-    process.exit(1)
-  }
-
-  return runCommandSync('git-cz')
+exports.setupGit = () => {
+  runGitCommand(['config', 'user.email', 'instructure-ui-ci@instructure.com'])
+  runGitCommand(['config', 'user.name', 'instructure-ui-ci'])
```

Comment:
this is a rather simplified version of the former one. can you please tell me more about this on the next standup?

Reply:
instui-dev@instructure.com